src/components/Search/SelectedRevisionItem.tsx

```diff
@@ -81,7 +81,11 @@ function SelectedRevisionItem({
         {isWarning && repository === 'try' && (
           <div className='warning-icon'>
             <Tooltip placement='top' title={warning}>
-              <WarningIcon fontSize='small' color='warning' />
+              <WarningIcon
+                fontSize='small'
+                color='warning'
+                titleAccess={warning}
```

Comment:
This adds a `title` to the icon, which is good for accessibility and makes it possible to target it from the test. The `title` in the `Tooltip` doesn't show up in the accessibility tree...